app/controllers/departments_controller.rb

```diff
@@ -40,7 +40,7 @@ def update
 
   def destroy
     @department.destroy
-    flash[:notice] = "Department was successfully destroyed."
+    flash[:notice] = I18n.t('department.destroy.success')
```

Comment:
Since you are in the `departments_controller` it's unnecessary to mention `department` everywhere in the localizations. It should be reordered in `.yml` file in order to have just `t('.destroy.success')`

Reply:
The same situation i with all other controllers.